packages/presentation/lib/screens/splash/splash_bloc.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:domain/usecases/delay_usecase.dart';
+import 'package:get_it/get_it.dart';
+import 'package:presentation/bloc/base/bloc.dart';
+import 'package:presentation/bloc/base/bloc_impl.dart';
+import 'package:presentation/navigation/app_navigator.dart';
+import 'package:presentation/screens/main/main_screen.dart';
+import 'package:presentation/screens/splash/data/splash_data.dart';
+
+abstract class SplashBloc implements Bloc<SplashData> {
+  factory SplashBloc() => _SplashBloc(GetIt.I.get<DelayUsecase>());
+}
+
+class _SplashBloc extends BlocImpl<SplashData> implements SplashBloc {
+  final AppNavigator _appNavigator = GetIt.I.get<AppNavigator>();
```

Comment:
move it to BlocImpl then u'll have appNavigator in each bloc by default

Reply:
Fixed.